docs/how-to/migrate-to-mas.md

```diff
@@ -69,49 +71,52 @@ juju run synapse/leader restore-backup backup-id=<backup-id>
 ```
 
 ## Synapse database migration
-Stop all services on the new and the existing synapse charms:
+Scale both synapse charms to 0 unit:
 ```
-juju ssh --container synapse server-mas/0 "pebble stop synapse; pebble stop synapse-mas; pebble stop stats-exporter"
-juju ssh --container synapse server/0 "pebble stop synapse; pebble stop stats-exporter"
+juju scale-application server-mas 0
+juju scale-application server 0
 ```
 
-Connect the new synapse charm to the database:
+Run `juju status` and save the IP address of the postgresql charm leader unit to an environment variable `DB_HOST`:
 ```
-juju ssh --container synapse server-mas/0 bash
+juju status
+export DB_HOST="<ip-of-the-postgresql-leader-unit>"
 ```
 
-The following commands assume that you are in the `synapse` container of the `server-mas` application.
+Get the password of the postgresql charm, then save the password to an environment variables `PGPASSWORD`:
 ```
-apt update
-apt install wget postgresql-client -y
-wget https://github.com/mikefarah/yq/releases/download/v4.44.3/yq_linux_amd64 -O /usr/bin/yq && chmod +x /usr/bin/yq
+juju run postgresql-k8s/leader get-password
+export PGPASSWORD=<output-of-the-above-command>
 ```
 
 Copy the original synapse charm database:
 ```
-DB_HOST=$(yq e '.database.args.host' /data/homeserver.yaml)
-DB_PORT=$(yq e '.database.args.port' /data/homeserver.yaml)
-DB_PASSWORD=$(yq e '.database.args.password' /data/homeserver.yaml)
-DB_USER=$(yq e '.database.args.user' /data/homeserver.yaml)
-PGPASSWORD=$DB_PASSWORD psql --host $DB_HOST --username $DB_USER --port $DB_PORT postgres -c 'drop database "server-mas";'
-PGPASSWORD=$DB_PASSWORD psql --host $DB_HOST --username $DB_USER --port $DB_PORT postgres -c "create database \"server-mas\" with template server owner $DB_USER;"
+PGPASSWORD=$PGPASSWORD psql --host $DB_HOST --username operator postgres -c 'drop database "server-mas";'
+PGPASSWORD=$PGPASSWORD psql --host $DB_HOST --username operator postgres -c "create database \"server-mas\" with template server;"
 ```
 
-In case where postgres gives you an error saying "ERROR:  source database "synapse" is being accessed by other users", drop all connections from the database:
 
-First log in to the database:
+## Migrate users to MAS
+### Update token for Draupnir
+If the old synapse charm has moderation enabled, we need to modify the corresponding access token so that it does not get ignored by `syn2mas`.
+
+Run the following SQL command on the old synapse charm's database:
 ```
-PGPASSWORD=$DB_PASSWORD psql --host $DB_HOST --username $DB_USER --port $DB_PORT postgres
+PGPASSWORD=$PGPASSWORD psql --host $DB_HOST --username operator server "update access_tokens set device_id='<device-id>' where token='<mjolnir-access-token>';"
```

Comment:
Device ID must exist in the devices table.

Reply:
thanks, I added a note